### PR TITLE
Basic functionality

### PR DESCRIPTION
**Pull request by golovnevs**
Hello teachers! I finished the functions for displaying the quiz and calculating the results of the quiz.  #6

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+      </a>
+      <div class="middle">
+        <div class="text">❤️</div>
+        </div>
+      </div>
+    </div>
+    `;
+  } else {
+      displayResult();
+    }
```
Comment (MrTim) [inline js/script.js]: indentation

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+let leftOutfit = 0;
+let rightOutfit = 1; 
+let steps = 0;
+let totalResult =0;
+
+function displayQuiz() {
+  const main = document.getElementById('main');
+  quizDiv = document.createElement("div");
+  quizDiv.setAttribute("class", "quiz");
+  if (steps <= outfits.length-2) {
```
Comment (MrTim) [inline js/script.js]: for better readability you can put spaces around `-` sign

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+    </div>
+    `;
+  } else {
+      displayResult();
+    }
+  voteRight = document.getElementById("rightPic");
+  voteRight.addEventListener('click', voteCounterRight);
+  voteLeft = document.getElementById("leftPic");
+  voteLeft.addEventListener('click', voteCounterLeft);
+  console.log(steps);
```
Comment (MrTim) [inline js/script.js]: console.log)

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+  rightClickCounter = 0;
+  liked.push(outfits[leftOutfit])
+  if (leftOutfit == 0 && rightOutfit == 1) {
+    rightOutfit = leftOutfit + 2;
+  } else if (leftClickCounter == 0) {
+    rightOutfit = leftOutfit + 1;
+  } else  {
+    rightOutfit++;
+  }
+    steps++;
```
Comment (MrTim) [inline js/script.js]: indentation?

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,410 @@
...
+  displayQuiz();
+};
+
+function displayResult() {
+  removeDuplicates();
+  leftOutfit = 0;
+  rightOutfit = 1;
+  steps = 0; 
+  table = document.getElementById('resultTable');
+  table.innerHTML = ` <h1>Here is the most liked outfits:</h1>
```
Comment (MrTim) [inline js/script.js]:
```js
table.innerHTML = ` <h1>Here is the most liked outfits:</h1>`;
```
same for next row

File: js/script.js [hunk 6]
```diff
@@ -0,0 +1,410 @@
...
+    <a>
+    <img src=${likedUniq[i].imgSrc}>
+    </a>
+    <div class="desc">${likedUniq[i].items}</div>
+    `;
+  table.appendChild(outfitCard);
+  } 
+};
+
+function removeDuplicates() {
```
Comment (MrTim) [inline js/script.js:397]:
```js
function removeDuplicates() {
  for (let i = 0; i < liked.length; i++) {
    if (liked[i] != liked[i+1]) {
      likedUniq.push(liked[i]);
    }
  }
};
```

Comment (MrTim) [reply js/script.js:397]: And you don't need to have globally available variable `likedUniq` (and by the way there is typo - unique).
So you can remove duplicates and return it as a result from that function.

**Review by MrTim — CHANGES_REQUESTED** [review]
Nice work! Left some comment. And please check the indentation and code structuring (functions and ifs)

Possible improvements - me as a user I would close this page after 10th click. Why? because I don't know how many steps are left and maybe it's a million. So maybe you can think of provide an info how many steps is there in the page before user starts. Just some suggestions :)

**Comment by MrTim** [comment]
@golovnevs oh, and there is a bug just before you display the results
`Uncaught TypeError: Cannot read property 'addEventListener' of null`. Please check that as well.

File: js/script.js (hunk 1 above)
Comment (golovnevs) [reply js/script.js]: Great idea about steps, thank you! I will try to make a status bar or something like that.

File: js/script.js (hunk 3 above)
Comment (golovnevs) [reply js/script.js]: Fixed :)

File: js/script.js (hunk 6 above)
Comment (golovnevs) [reply js/script.js:397]: Thank you, that would be better.

File: js/script.js (hunk 2 above)
Comment (golovnevs) [reply js/script.js]: Fixed.

File: js/script.js (hunk 4 above)
Comment (golovnevs) [reply js/script.js]: Fixed.

File: js/script.js (hunk 6 above)
Comment (golovnevs) [reply js/script.js:397]: Fixed.

**Comment by golovnevs** [comment]
@golovnevs oh, and there is a bug just before you display the results
Uncaught TypeError: Cannot read property 'addEventListener' of null. Please check that as well.

Fixed with your help :)

File: js/script.js (hunk 6 above)
Comment (golovnevs) [reply js/script.js:405]: Fixed.

File: js/script.js (hunk 5 above)
Comment (golovnevs) [reply js/script.js]: Fixed.

File: js/script.js (hunk 1 above)
Comment (golovnevs) [reply js/script.js]: Problem with Indentation - fixed

Comment (golovnevs) [reply js/script.js]: Step counter added. Next step - to create animated status bar.

File: js/script.js [hunk 7]
```diff
@@ -293,19 +286,30 @@ let links = [
 ];
 
 let liked = [];
-let likedUniq = [];
 let leftOutfit = 0;
 let rightOutfit = 1; 
-let steps = 0;
+let steps = 1;
 let totalResult =0;
```
Comment (MrTim) [inline js/script.js:292]: space after `=` :)

File: js/script.js [hunk 8]
```diff
@@ -293,19 +286,30 @@ let links = [
...
+const startBtn = document.getElementById('start');
+startBtn.addEventListener('click', displayContent);
+let leftClickCounter = 0;
+let rightClickCounter = 0;
+
+function displayContent() {
+  if (steps <= 20) {
+    displayQuiz();
+  } else {
+      displayResult();
```
Comment (MrTim) [inline js/script.js:302]: hm...indentation again different for that line

**Review by MrTim — APPROVED** [review]
Good, please merge it!

File: js/script.js (hunk 7 above)
Comment (golovnevs) [reply js/script.js:292]: Fixed)

**Pull request merged**